com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/tester.py

```diff
@@ -217,6 +218,21 @@ def setup_streaming_analytics(test, service_name=None, force_remote_build=False)
         if force_remote_build:
             test.test_config['topology.forceRemoteBuild'] = True
 
+        vcap_services = os.environ.get('VCAP_SERVICES')
+        if not vcap_services.startswith('/'):
+            #vcap services is json
+            vcap_services = json.loads(vcap_services)
+        else:
+            with open(vcap_services) as fh:
+                vcap_services = json.load(fh)
+
+        test.is_v2 = False
```

Comment:
This (`is_v2`) is a really generic name to write into someone's test case object. I also don't see it documented anywhere.

Is there a use case to have it exposed, or is this really only for the special case of the topology tests which need to run against both?

Reply:
Yes, it's "only" for that special case. While it should be documented, it does seem that a handful of tests depend on the service version.

Reply:
Why not just have it in topology unittests, and not the tester.